chapters/methodology/theory.tex

```diff
@@ -127,15 +149,18 @@ \subsection{The $\Omega$ Methods}
                            d\Omega}.
 \label{eq:omega_basic}
 \end{equation}
-%
+% should the d \Omega have a hat? I'm not sure...
+
 The $\Omega$-flux is a hybridization of the adjoint scalar flux and the contributon
 flux. It is both a normalized contributon flux and a forward-weighted
 adjoint flux. Theoretically, it should inherit some of the advantages of
 each of the traditional
 adjoint and the contributon fluxes. Because it maintains dimensionality of the
 traditional adjoint scalar flux, it can be used in place of the standard adjoint
 scalar flux in both
-CADIS and FW-CADIS variance reduction parameter generation.
+CADIS and FW-CADIS variance reduction parameter generation. 
+This means that nothing about the CADIS and FW-CADIS implementations need to 
```

Comment:
I'm a little hesitant to add that nothing about their implementations need to change and that this is straightforward to adopt, because 
- it could be interpreted as this is trivially easy to do. It wasn't. 
- cadis had to be modified to run a forward calculation in addition to the adjoint calculation
- the existing implementations of CADIS and FW-CADIS did have to be changed, but the infrastructure used to generate the variance reduction parameters didn't have to change
I'll play around with wording to find a compromise here. 
What do you think?